PokemonSearch/PokemonSearch/SearchTableViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SearchTableViewController.swift
+//  PokemonSearch
+//
+//  Created by Jocelyn Stuart on 1/25/19.
+//  Copyright © 2019 JS. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewController: UITableViewController, UISearchBarDelegate {
```

Comment:
For custom cell delegation this class needs to adapt the protocol
```suggestion
class SearchTableViewController: UITableViewController, UISearchBarDelegate, SearchTableViewCellDelegate {
```